pages/landing.tmpl

```diff
@@ -73,6 +78,22 @@ form dd {
 
 	<div{{if not .OpenRegistration}} style="padding: 2em 0;"{{end}}>
 		{{ if .OpenRegistration }}
+		{{if .DisablePasswordAuth}}
```

Comment:
With this logic, admins won't be able to allow **both** password and OAuth account creation. So I think we'll want to repeat the logic from `login.tmpl` here, and simply hide the password-based signup form when `DisablePasswordAuth == true`, instead of making the signup methods mutually exclusive.